mesonbuild/msetup.py

```diff
@@ -162,15 +162,10 @@ def validate_dirs(self) -> T.Tuple[str, str]:
         has_partial_build = os.path.isdir(priv_dir)
         if has_valid_build:
             if not self.options.reconfigure and not self.options.wipe:
-                print('Directory already configured.\n\n'
-                      'Just run your build command (e.g. ninja) and Meson will regenerate as necessary.\n'
-                      'Run "meson setup --reconfigure to force Meson to regenerate.\n\n'
-                      'If build failures persist, run "meson setup --wipe" to rebuild from scratch\n'
-                      'using the same options as passed when configuring the build.')
                 if self.options.cmd_line_options:
-                    from . import mconf
-                    raise SystemExit(mconf.run_impl(self.options, build_dir))
-                raise SystemExit(0)
+                    print('Directory already configured, applying new settings.')
+                else:
+                    print('Directory already configured.')
```

Comment:
Why drop the `mconf.run_impl()` call? Shouldn't this change be just about changing the logging but keeping the implementation the same?